pkg/addons/data/DaemonSet.apps/kube-service-catalog/controller-manager.yaml

```diff
@@ -17,31 +18,47 @@ spec:
       - args:
         - controller-manager
         - --secure-port
-        - "8080"
+        - "6443"
         - -v
         - "3"
         - --leader-election-namespace
         - kube-service-catalog
         - --leader-elect-resource-lock
         - configmaps
+        - --cluster-id-configmap-namespace=kube-service-catalog
         - --broker-relist-interval
         - 5m
         - --feature-gates
         - OriginatingIdentity=true
+        - --feature-gates
+        - AsyncBindingOperations=true
+        - --feature-gates
+        - NamespacedServiceBroker=true
         command:
         - /usr/bin/service-catalog
+        env:
+        - name: K8S_NAMESPACE
+          valueFrom:
+            fieldRef:
+              apiVersion: v1
+              fieldPath: metadata.namespace
         image: '*** GENERATED ***'
         name: controller-manager
         ports:
-        - containerPort: 8080
+        - containerPort: 6443
         volumeMounts:
         - mountPath: /var/run/kubernetes-service-catalog
           name: service-catalog-ssl
           readOnly: true
       nodeSelector:
-        node-role.kubernetes.io/infra: "true"
+        node-role.kubernetes.io/master: "true"
```

Comment:
Why master? We should continue running these in infra I think.

Reply:
follows upstream

Reply:
agreed to follow upstream due etcd secret case with apiserver